packages/react/src/components/Comments/ThreadsSidebar.tsx

```diff
@@ -4,40 +4,46 @@ import { Thread } from "./Thread.js";
 import { useThreads } from "./useThreads.js";
 import { ThreadData } from "@blocknote/core/comments";
 import { BlockNoteEditor } from "@blocknote/core";
+import { useEffect, useMemo } from "react";
```

Comment:
I see you still need to resolve my previous comments to this file - so for now I'll skip re-reviewing :)